tests/integrationtest/l4_l7_namespace_shard_nodeport_test.go

```diff
@@ -353,3 +353,63 @@ func TestNoHostIngressInNodePort(t *testing.T) {
 
 	TearDownTestForIngressInNodePortMode(t, model_Name)
 }
+
+func TestNoHostIngressInNodePortWithMultiTenantEnabled(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	// enable tenants per cluster
+	SetAkoTenant()
+	defer ResetAkoTenant()
```

Comment:
Is the resetTenant really required? in a running AKO the tenant would be always set to the Ako tenant?

Reply:
Yes, Since other tests are relying on the env variable this ensures that other tests run with admin tenent only.

Reply:
Okay makes sense.